# Description: Change a few default mount options for compatibility to UTF8 and previous Ubuntu versions.
--- gnome-mount-0.8~svn20080225/gnome-mount.schemas.in	2007-08-01 02:49:20.000000000 +0200
+++ gnome-mount-0.8~svn20080225.new/gnome-mount.schemas.in	2008-03-05 15:30:05.000000000 +0100
@@ -6,7 +6,7 @@
         <applyto>/system/storage/default_options/vfat/mount_options</applyto>
         <type>list</type>
         <list_type>string</list_type>
-        <default>[shortname=lower,uid=]</default>
+        <default>[shortname=mixed,uid=,utf8,umask=077,exec,flush]</default>
         <locale name="C">
           <short>Default mount options for vfat fs</short>
           <long>A list of default mount options for volumes formatted with the vfat file system.</long>
@@ -18,7 +18,7 @@
         <applyto>/system/storage/default_options/iso9660/mount_options</applyto>
         <type>list</type>
         <list_type>string</list_type>
-        <default>[uid=]</default>
+        <default>[uid=,utf8,exec]</default>
         <locale name="C">
           <short>Default mount options for iso9660 fs</short>
           <long>A list of default mount options for volumes formatted with the iso9660 file system.</long>
@@ -30,7 +30,7 @@
         <applyto>/system/storage/default_options/udf/mount_options</applyto>
         <type>list</type>
         <list_type>string</list_type>
-        <default>[uid=]</default>
+        <default>[uid=,exec]</default>
         <locale name="C">
           <short>Default mount options for udf fs</short>
           <long>A list of default mount options for volumes formatted with the udf file system.</long>
@@ -42,7 +42,7 @@
         <applyto>/system/storage/default_options/ntfs/mount_options</applyto>
         <type>list</type>
         <list_type>string</list_type>
-        <default>[umask=222]</default>
+        <default>[umask=222,utf8,exec]</default>
         <locale name="C">
           <short>Default mount options for ntfs fs</short>
           <long>A list of default mount options for volumes formatted with the ntfs file system.</long>
@@ -64,13 +64,23 @@
         <applyto>/system/storage/default_options/ntfs-3g/mount_options</applyto>
         <type>list</type>
         <list_type>string</list_type>
-        <default>[locale=]</default>
+        <default>[locale=,exec]</default>
         <locale name="C">
           <short>Default mount options for ntfs-3g fs</short>
           <long>A list of default mount options for volumes formatted with the ntfs file system using ntfs-3g.</long>
         </locale>
       </schema>
 
-
+      <schema>
+        <key>/schemas/system/storage/default_options/hfs/mount_options</key>
+        <applyto>/system/storage/default_options/hfs/mount_options</applyto>
+        <type>list</type>
+        <list_type>string</list_type>
+        <default>[uid=,]</default>
+        <locale name="C">
+          <short>Default mount options for hfs fs</short>
+          <long>A list of default mount options for volumes formatted with the hfs file system.</long>
+        </locale>
+      </schema>
   </schemalist>
 </gconfschemafile>
